tools/kompos/src/visualizations.ts

```diff
@@ -15,7 +15,7 @@ var width = 360,
 
 function hasSelfSends(actorId, messages) {
   for (var i in messages) {
-    if (messages[i].sender == "actorId") {
+    if (messages[i].sender === "actorId") {
       return true;
     }
   }
```

Comment:
`visualizations.ts` is for graphical things, the actor history. While `view.ts` is for normal UI elements.